Vagrantfile

```diff
@@ -147,6 +147,6 @@ def post_up_message
   msg
 end
 
-def remote_site_path(site_name)
-  "/srv/www/#{site_name}/current"
+def remote_site_path(site_name, site)
+  "/srv/www/#{site_name}/#{current_path = site['current_path'] || 'current'}"
```

Comment:
no need for assignment here. just: `"/srv/www/#{site_name}/#{site['current_path'] || 'current'}"`